### ISSUE TITLE
[Feature Request] Add accessible global or per-site zoom (scale) option

### ISSUE THREAD

**Issue by Peachy6769** [nonbili/Nora#288]
Either a global (in app settings) or per-site (in toolbar menu) option for web zoom/scale should be implemented to increase accessibility for those who require larger screen elements for legibility.
It would function the same way "zoom" does in a browser:
![Screenshot_20260615-222959.png](https://github.com/user-attachments/assets/97187cde-af04-4031-84b5-f9300dcd80a4)
